x/commitment/keeper/commitments.go

```diff
@@ -162,6 +162,7 @@ func (k Keeper) BurnEdenBoost(ctx sdk.Context, creator sdk.AccAddress, denom str
 	if err != nil {
 		return err // never happens
 	}
+	k.SetCommitments(ctx, commitments)
```

Comment:
We don't need to call hooks: hooks.CommitmentChanged
?

Reply:
yes, updated, thanks